conformance/utils/flags/flags.go

```diff
@@ -27,4 +27,6 @@ var (
 	GatewayClassName     = flag.String("gateway-class", "gateway-conformance", "Name of GatewayClass to use for tests")
 	ShowDebug            = flag.Bool("debug", false, "Whether to print debug logs")
 	CleanupBaseResources = flag.Bool("cleanup-base-resources", true, "Whether to cleanup base test resources after the run")
+	SupportedFeatures    = flag.String("supported-features", "SupportReferenceGrant", "Supported features included in conformance tests suites")
+	ExemptFeatures       = flag.String("exempt-features", "", "Exempt Features excluded from conformance tests suites")
```

Comment:
Instead of using `flag.String` and then having helper parsing functions we can use `flag.StringSlice`.

Reply:
To not break the current behavior, let's keep the default value of `SupportedFeatures` as `[]suite.SupportedFeature{suite.SupportReferenceGrant}`

Reply:
> Instead of using `flag.String` and then having helper parsing functions we can use `flag.StringSlice`.

Sorry, I didn't find `flag.StringSlice` in flag packge https://pkg.go.dev/flag.

> To not break the current behavior, let's keep the default value of SupportedFeatures as []suite.SupportedFeature{suite.SupportReferenceGrant}

The helper parsing function will convert the value `SupportedFeatures` to `[]suite.SupportedFeature{suite.SupportReferenceGrant}`, which doesn't break current behavior and makes the arguments looks more concise.

Reply:
My bad, I though we were using [pflag](https://pkg.go.dev/github.com/spf13/pflag) 🤦

Reply:
It's fine, haha. I hope it could have a `StringSlice` function, but it doesn't, I have to use another helper parsing function.